r/R/cluster.R

```diff
@@ -79,19 +110,52 @@ getClusters <- function(type, resolution, data) {
 #' @return boolean indicating if SNN Graph object exists
 #'
 getSNNiGraph <- function(data, active.reduction) {
-
   # check to see if we already have Seurat SNN Graph object
   snn_name <- paste0(data@active.assay, "_snn")
 
   # if doesn't exist, run SNN
-  if (!snn_name %in% names(data)) data <- Seurat::FindNeighbors(data, reduction = active.reduction)
+  if (!snn_name %in% names(data)) {
+    data <- Seurat::FindNeighbors(data, reduction = active.reduction)
+  }
 
   # convert Seurat Graph object to igraph
   # similar to https://github.com/joshpeters/westerlund/blob/46609a68855d64ed06f436a6e2628578248d3237/R/functions.R#L85
-  adj_matrix <- Matrix::Matrix(as.matrix(data@graphs[[snn_name]]), sparse = TRUE)
+  adj_matrix <-
+    Matrix::Matrix(as.matrix(data@graphs[[snn_name]]), sparse = TRUE)
   g <- igraph::graph_from_adjacency_matrix(adj_matrix,
     mode = "undirected",
     weighted = TRUE
   )
   return(g)
 }
+
+format_cell_sets_object <-
+  function(cell_sets, clustering_method, color_pool) {
+    name <- paste0(clustering_method, " clusters")
+
+    # careful with capital l on type for the key.
```

Comment:
I don't understand this comment -- where are we supposed to be careful and why?

Reply:
```suggestion
```